modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -96,7 +96,7 @@ The query frontend component is responsible for sharding the search space for an
 |type: boolean
 
 |`jaegerQuery.monitorTab.prometheusEndpoint`
-|The endpoint to the Prometheus instance that contains the span rate, error, and duration (RED) metrics. For example, `+https://thanos-querier.openshift-monitoring.svc.cluster.local:9091+`.
+|The endpoint to the Prometheus instance that contains the span rate, error, and duration (RED) metrics. For example, `+https://thanos-querier.openshift-monitoring.svc.cluster.local:9092+`.
```

Comment:
I cannot see this change in the preview. Would you be OK to provide a link?

Reply:
Ah, foound it here: https://91498--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring